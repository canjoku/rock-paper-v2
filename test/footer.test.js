import Footer from '../components/footer'

test('<Footer /> renders correctly', () => {
  const component = shallow(<Footer />)
  expect(component).toMatchSnapshot()
})
