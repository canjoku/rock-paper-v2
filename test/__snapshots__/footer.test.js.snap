// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly 1`] = `
<div>
  <ul>
    <li>
      <a
        href="#"
      >
        Footer Link
      </a>
    </li>
  </ul>
</div>
`;
